src/Alert.php

```diff
@@ -111,7 +113,17 @@ public static function dropdown($options = [])
         }
 
         $times[Entity::CONFIG_NEVER]  = __('Never');
+        if ($p['show_hours']) {
+            for ($i = 1; $i <= 24; $i++) {
+                $times[$i] = sprintf(__('%1$s hours'), $i);
```

Comment:
```suggestion
                $times[$i * HOUR_TIMESTAMP] = sprintf(_('Every %1$s hours'), $i);
```

Reply:
Maybe you could also have a `$times[HOUR_TIMESTAMP] = __('Each hour');` and start your loop at `$i = 2` to have a better wording.